tests/framework/checks/fails.go

```diff
@@ -46,9 +46,8 @@ func SkipIfPrometheusRuleIsNotEnabled(virtClient kubecli.KubevirtClient) {
 	}
 }
 
-// Deprecated: SkipIfRunningOnKindInfra should be converted to check & fail
-func SkipIfRunningOnKindInfra(message string) {
+func FailTestIfRunningOnKindInfra(message string) {
```

Comment:
Since there are still skip functions here, please consider deferring the file rename.